grammars.yml

```diff
@@ -1057,6 +1057,11 @@ vendor/grammars/verilog.tmbundle:
 - source.verilog
 vendor/grammars/vhdl:
 - source.vhdl
+vendor/grammars/vsc-ember-syntax:
+- inline.hbs
+- inline.template
+- source.gjs
+- source.gts
```

Comment:
I manually removed `text.html.handlebars` from the `vsc-ember-syntax` in grammars.yml to (hopefully) avoid conficting with the existing Handlebars grammar.

Reply:
Good thinking -- because ember doesn't use handlebars anyway

Reply:
Right and we don't want to disrupt users of `.hbs`/Handlebars (ember or otherwise) with this change.